### DISCUSSION TITLE
Possible to connect data feed to Meeting Guide App and CER website?

### DISCUSSION THREAD

**Discussion by aamonterey** [code4recovery/12-step-meeting-list#908] [Q&A] [answered]
I'm assisting a WebTeam in Portugal to setup a new English speaking website at: [aaportugal.net](https://aaportugal.net/). Their meetings are currently published on the [CER](https://alcoholics-anonymous.eu/) website which has a data feed to the Meeting Guide App.

The new site in Portugal will be the official listing of English speaking meetings, and besides setting up a data feed to the Meeting Guide App, they want it to feed the CER website. Is it possible to have the data feed from aaportugal.net go to both Meeting Guide and CER? If so, would the CER feed need to be revised so Meeting Guide doesn't list the meetings twice?

Would also appreciate being directed to setting up the JSON data feed.

**Comment by joshreisner** [comment, marked as answer]
hi @aamonterey - i've transferred the issue here. 

they should be able to do this - the steps roughly are:

1. make sure all the meetings are managed locally in 12 Step Meeting List on AA Portugal (and not being imported)
2. create an "authorized app" data feed (or open up sharing) and provide it CER - [more instructions on that here](https://meetingguide.helpdocs.io/article/tjn0po3zco-providing-meeting-listings-to-another-entity)
3. CER deletes its local listings for Portugal, and adds AA Portugal's feed to its imports (they are familiar with this process)
4. provide the data feed to GSO (appsupport@aa.org) and tell them that all conflicts should resolve to AA Portugal, because they are the originating website

**Reply by aamonterey** [reply]
Thanks, Josh! Much appreciated.
